src/rt/backtrace/elf.d

```diff
@@ -45,21 +60,16 @@ struct Image
 
     const(ubyte)[] getDebugLineSectionData()
     {
-        auto stringSectionHeader = ElfSectionHeader(&file, file.ehdr.e_shstrndx);
-        auto stringSection = ElfSection(&file, &stringSectionHeader);
+        ElfSectionHeader dbgSectionHeader;
+        if (!file.findSectionHeaderByName(".debug_line", dbgSectionHeader))
+            return null;
 
-        auto dbgSectionIndex = findSectionByName(&file, &stringSection, ".debug_line");
-        if (dbgSectionIndex != -1)
-        {
-            auto dbgSectionHeader = ElfSectionHeader(&file, dbgSectionIndex);
-            // we don't support compressed debug sections
-            if ((dbgSectionHeader.shdr.sh_flags & SHF_COMPRESSED) != 0)
-                return null;
-            // debug_line section found and loaded
-            return ElfSection(&file, &dbgSectionHeader);
-        }
+        // we don't support compressed debug sections
+        if ((dbgSectionHeader.shdr.sh_flags & SHF_COMPRESSED) != 0)
+            return null;
 
-        return null;
+        // debug_line section found and loaded
+        return cast(const(ubyte)[]) ElfSection(file, dbgSectionHeader).get();
```

Comment:
This should be blatantly wrong, unmapping the section data right before returning the slice. I wonder why the original code seems to be working...